src/main/webapp/find-events.html

```diff
@@ -64,54 +64,57 @@ <h1 id="find-events-title" tabindex="0">find events</h1>
           </div>
         </div>
         <div id="all-posts">
-          <button id="create-post-button" type="button" tabindex="0" aria-label="create a post">create a post</button>
+          <button aria-label="create a post" id="create-post-button" tabindex="0" type="button">create a post</button>
```

Comment:
I believe `aria-labels` are only needed if there isn't visible/screen-reader accessible text (see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute), but lemme know if I'm missing something here!

Reply:
So for the case of using placeholder text as the label (like in the modal), `aria-label` is helpful because screenreaders don't always read placeholders, but this button does have normal containing text so I think it doesn't need it

Reply:
Good point!! I'll remove it